test/TestApps/PushNotificationsDemoPackage/PushNotificationsDemoPackage.wapproj

```diff
@@ -42,7 +42,13 @@
     <EntryPointProjectUniqueName>..\PushNotificationsDemoApp\PushNotificationsDemoApp.vcxproj</EntryPointProjectUniqueName>
     <GenerateAppxPackageOnBuild>true</GenerateAppxPackageOnBuild>
     <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
-    <PackageCertificateKeyFile>$(SolutionDir)temp\MSTest.pfx</PackageCertificateKeyFile>
+    <PackageCertificateKeyFile>$(SolutionDir).user\winappsdk.certificate.test.pfx</PackageCertificateKeyFile>
```

Comment:
Please remove the usage of this variable everywhere in your change.  SolutionDir is a terrible property especially in a tree that has more than one solution file (ours is one of those).  Use RepoRoot and full paths whereever possible.

Reply:
Agreed. I only discovered RepoRoot recently. Updates coming...